packages/replay/src/bin.ts

```diff
@@ -86,6 +91,7 @@ commandWithGlobalOptions("launch [url]")
 commandWithGlobalOptions("record [url]")
   .description("Launch the replay browser and start recording")
   .option("-b, --browser <browser>", "Browser to launch", "chromium")
+  .option("-a --browser-args <args>", "Browser arguments", "")
```

Comment:
re API choice - next is accepting a similar thing through `NODE_OPTIONS` env variable: https://nextjs.org/docs/pages/api-reference/next-cli . I don't have any particular opinion why it would be better though so I'm OK with the current solution

Reply:
Ok, I educated myself a little bit and it seems that it's just a standard env variable for node and it works in a very similar way as CLI arguments for node (you can pass both using both ways). I can't find any env variable like this for Chrome.

So, I think, the current proposed solution is more than fine and I would just go with it. However, I probably wouldn't allow this to be used with `-a`. This particular command *already* comes with the support for `--attach` and `-a` could be easily mistaken as a shortcut for that option.
```suggestion
  .option("--browser-args <args>", "Browser arguments", "")
```